### ISSUE TITLE
[ISSUE] setting `secrets.credentialsSecretName` doesn't do anything

### ISSUE THREAD

**Issue by ilovemysillybanana** [timescale/helm-charts#372]
**Describe the bug**
Setting the value `secrets.credentialsSecretName`, the deployment fails because it's still expecting there to be a secret by the default name and gives this error: `Error: secret "timescaledb-single-credentials" not found`

**To Reproduce**
Install the helm chart with default values, set the `secrets.credentialsSecretName` to the name of a secret.

**Expected behavior**
I would expect it to find the existing secret in it's namespace but it does not. Looking into the code, it doesn't look like it does anything if it sees that `secrets.credentialsSecretName` is not empty quotes `""`.

**Deployment**
- What is in your `values.yaml`
```yaml
  secrets:
    # This map should contain environment variables that influence Patroni,
    # for example PATRONI_SUPERUSER_PASSWORD or PATRONI_REPLICATION_PASSWORD
    # https://patroni.readthedocs.io/en/latest/ENVIRONMENT.html#postgresql
    # credentials:
    #  PATRONI_SUPERUSER_PASSWORD: ""
    #  PATRONI_REPLICATION_PASSWORD: ""
    #  PATRONI_admin_PASSWORD: ""

    # Selector used to provision your own Secret containing patroni configuration details
    # This is mutually exclusive with `credentials` option and takes precedence over it.
    # WARNING: Use this option with caution  
    credentialsSecretName: "timescaledb-credentials"
```
- What version of the Chart are you using? `0.11.0`
- What is your Kubernetes Environment (for exampe: GKE, EKS, minikube, microk8s) AWS EKS

**Comment by anthonycorbacho** [comment]
I am having the same issue, is there a way around it?

**Comment by paulfantom** [comment]
Fixed with #373. Please reopen if you find that the linked PR did not resolve your issue.
